docs/advanced/testing.md

```diff
@@ -184,10 +233,10 @@ import { Provider } from 'urql';
 import { makeSubject } from 'wonka';
 import { MyComponent } from './MyComponent';
 
-const [stream, pushResponse] = makeSubject();
+const { source: stream, next: pushResponse } = makeSubject();
```

Comment:
I also try to use this code for real and I got an error from this line (`(0 , _wonka.makeSubject) is not a function or its return value is not iterable`). I was able to get it work by doing that. Seems that the `makeSubject` returns a object instead of a tuple.

Reply:
Yes, that's outdated code that hasn't been fixed up. Nice catch!